exercises/tree-building/TreeBuildingTest.cs

```diff
@@ -0,0 +1,228 @@
+﻿using NUnit.Framework;
+
+public class TreeBuildingTest
+{
+    [Test]
+    public void One_node()
+    {
+        var records = new[]
+        {
+            new TreeBuildingRecord { RecordId = 0, ParentId = 0 }
+        };
+
+        var tree = TreeBuilder.BuildTree(records);
+
+        AssertTreeIsLeaf(tree, id: 0);
+    }
+
+    [Test]
```

Comment:
Well, that is on purpose, as the refactoring exercises already have a working implementation. In fact, the refactoring should be done whilst keeping all tests passing.